.github/workflows/i18n-consistency-checker.yaml

```diff
@@ -17,88 +17,69 @@ jobs:
+              result=$(git diff "${parent_hash}" HEAD "$file" | sed '1,4 s/^/# /')
+        
+              # Append to the issue.md file
+              cat <<- EOM >> issue.md
+          <details><summary><b>$content_title</b> ($file)</summary>
           
-              result=$(echo "$(git diff ${parent_hash} HEAD $file)" | sed '1,4 s/^/# /')
-              echo -e "$result"
-          
-              # Add the contents to the issue.md file
-              issue+="<details><summary><b>$content_title</b> ($file)</summary>\\n\\n"
-              issue+="- Original File(en): [$file](https://github.com/$GITHUB_REPOSITORY/blob/master/$file)\\n"
-              issue+="- Translated File(${{matrix.language}}): [$i18n_filename](https://github.com/$GITHUB_REPOSITORY/blob/master/$i18n_filename)\\n"
-              issue+="- [Diff on GitHub](https://github.com/yuhattor/innersourcecommons.org/compare/$i18n_last_update_hash...$original_last_update_hash)\\n"
-              issue+="- Days since overdue updates: $days_since_overdue_updates days\\n"
-              issue+="\`\`\`diff\\n"
-              issue+="$result"
-              issue+="\\n\`\`\`\\n"
-              issue+="</details>\\n"
-              echo -e "$issue" >> issue.md
-              issue=""
+          For more information, please compare [the original file(en)](https://github.com/$GITHUB_REPOSITORY/blob/master/$file) with [the translated file](https://github.com/$GITHUB_REPOSITORY/blob/master/$i18n_filename). You can view [the differences](https://github.com/$GITHUB_REPOSITORY/compare/$i18n_last_update_hash...$original_last_update_hash) on GitHub. The number of days since overdue updates is **$days_since_overdue_updates** days.
+              
+          \`\`\`diff
+          $result
+          \`\`\`
+          </details>
+          EOM
             fi
           done
-      - name: Create Issue
-        run: | 
-          # Declare the flags
-          declare -A flags=(
-            ["ja"]="🇯🇵 Japanese"
-            ["zh"]="🇨🇳 Chinese"
-          )
-
-          # Set the issue title
-          issue_title="${flags[${{matrix.language}}]}: Content Consistency Issue"
-
+        
           # Get the existing issue ID
-          existing_issue_id=$(gh issue list -S "state:open type:issue title:$issue_title" | cut -f1)
-
-          # If the issue.md file exists, create a new issue or comment on the existing issue
+          existing_issue_id=$(gh issue list -S "is:issue is:open $issue_title" | cut -f1)
+          echo "existing_issue_id: $existing_issue_id"
+          # Create a new issue or comment on the existing one
           if [ -f issue.md ]; then
             if expr "$existing_issue_id" : '^[0-9]*$' >/dev/null; then
               gh issue comment "$existing_issue_id" -F issue.md -R $GITHUB_REPOSITORY
             else
               gh issue create -t "$issue_title" -F issue.md -R $GITHUB_REPOSITORY -l "Type - Translation"
             fi
           fi
```

Comment:
```suggestion
          fi
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

I guess we still need it...
<img width="860" alt="image" src="https://github.com/InnerSourceCommons/InnerSourcePatterns/assets/15963767/ac5bc97f-3b3d-44b5-9a61-6e6364f54bbd">